### ISSUE TITLE
Add Handbook as Labs project

### ISSUE THREAD

**Issue by danfowler** [okfn/okfn.github.com#460]
https://github.com/okfn/handbook

**Comment by gsilvapt** [comment]
Not sure what I should here. Would you mind explaining, @danfowler ?

**Comment by danfowler** [comment]
Updating and elaborating on this page: http://okfnlabs.org/projects/datapatterns/

Adding yourself as a contributor, and perhaps explaining how people can usefully contribute 😄

**Comment by gsilvapt** [comment]
Thanks! I need to find that repo and then will update it. Not sure if I am the right person to tell people how to usefully contribute to Labs but will give it a try. :smile:

**Comment by danfowler** [comment]
@gsilvapt well, how to contribute to the handbook at least 👍 

e.g. that it's hosted on GitHub (Pages) and so on.

**Comment by gsilvapt** [comment]
Wait, this is confusing. This is the project I have been working on with Rufus.

The newest version is uploaded here: http://okfnlabs.org/handbook/

datapatterns.org is not used anymore. The page itself, I am not sure where that okfnlabs.org/projects/datapatterns/ repository is, to correct those issues. Nevertheless, the project name *Data Patterns* is no longer used, instead we used **Handbook**.

**Comment by danfowler** [comment]
Yeah, I agree.  What used to be the "datapatterns" repo is now the "handbook" repo (https://github.com/okfn/datapatterns redirects to https://github.com/okfn/handbook).  Given that it is the same project essentially, I'm happy to just redirect the datapatterns.org domain name to http://okfnlabs.org/handbook/.

**Comment by rgrp** [comment]
@danfowler yes, please let's redirect datapatterns.org to okfnlabs.org/handbook/!

**Comment by danfowler** [comment]
We can use an internal redirect in Jekyll.

**Comment by rgrp** [comment]
@danfowler personally i'd boot a new project and mark datapatterns as deprecated as worth emphasizing the change.
